CHANGELOG

```diff
@@ -1,3 +1,13 @@
+21 January 2015 - Release 0.14.4
```

Comment:
I was freaked out by the date, it looked as if you plan to release at 21 January. I often write `???` there if the date is not known.